pyanaconda/modules/storage/partitioning/automatic/utils.py

```diff
@@ -255,8 +255,7 @@ def schedule_implicit_partitions(storage, disks, scheme, encrypted=False, luks_f
 
     return devs
 
-
-def get_default_partitioning():
+def get_default_partitioning(scheme=None):
     """Get the default partitioning requests.
 
     :return: a list of partitioning specs
```

Comment:
Please add the new parameter to docstring.

Reply:
Done.